pki/services.go

```diff
@@ -588,3 +597,19 @@ func GenerateRKEServicesCSRs(ctx context.Context, certs map[string]CertificatePK
 	}
 	return nil
 }
+
+func deleteUnusedCerts(certs map[string]CertificatePKI, certName string, hosts []*hosts.Host) {
```

Comment:
Can we add some logging here to indicate what we are doing

Reply:
Done